src/UiPath.Workflow/Microsoft/VisualBasic/Activities/VisualBasicReference.cs

```diff
@@ -12,17 +12,14 @@
 using System.Diagnostics;
 using System.Linq.Expressions;
 using System.Windows.Markup;
-using ActivityContext = System.Activities.ActivityContext;
 
 namespace Microsoft.VisualBasic.Activities;
 
 [DebuggerStepThrough]
 public sealed class VisualBasicReference<TResult> : CodeActivity<Location<TResult>>, IValueSerializableExpression,
     IExpressionContainer, ITextExpression
 {
-    private Expression<Func<ActivityContext, TResult>> _expressionTree;
```

Comment:
Removing the expression tree is very interesting. I had looked before to see if there were any dependencies on it and it seemed there were. But if this expression tree can be removed going forward I'm all for it.

Reply:
considering we set RequiresCompilation = true for VB too, this does not make any sense now. The only difference is now you need to compile the workflow, before you can access the expression tree of an ITextExpression (which was already the case for C#). Currently we need to have SWeb on board, as they seem to be using GetExpressionTree.